libcxx/test/libcxx/containers/container.adaptors/flat.set/version.pass.cpp

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Let's make this a `.compile.pass.cpp` test.

Reply:
All the existing `version.pass.cpp` tests are not compile tests (e.g. `libcxx/test/libcxx/containers/sequences/vector/version.pass.cpp`). Do you want me to make that patch, or will you?

Reply:
On a similar subject, #11.